apps/worker/src/app/workflow/workflow.module.ts

```diff
@@ -59,6 +62,7 @@ import {
   WebhookFilterBackoffStrategy,
 } from './usecases';
 import { MetricQueueService } from './services/metric-queue.service';
+import { ProcessTenant } from '@novu/application-generic/build/main/usecases/process-tenant';
```

Comment:
@ainouzgali this import slipped through the cracks. Could you please fix it in `next`? 🙏🏻